lib/async_core.dart

```diff
@@ -62,6 +62,7 @@ Future<WebDriver> createDriver(
   final session = await client.send(
```

Comment:
nit: this should have probably been using `try {} finally {}` to make sure that client is closed even if `send` causes an exception. Otherwise you might end up with the same 15s timeout. 

Not that it matters for typical usage - but it is cleaner that way.